### ISSUE TITLE
AnnotationView Crash

### ISSUE THREAD

**Issue by joeabouassi** [Instabug/android-sample#60]
Instabug 4.2.8 reported this crash in the SDK itself: 

> width and height must be > 0
> AnnotationView.java:727

```
width and height must be > 0
    at android.graphics.Bitmap.createBitmap (Bitmap.java:967)
    at android.graphics.Bitmap.createBitmap (Bitmap.java:946)
    at android.graphics.Bitmap.createBitmap (Bitmap.java:913)
    at com.instabug.library.view.annotation.AnnotationView.a (AnnotationView.java:727)
    at com.instabug.library.view.annotation.AnnotationView.b (AnnotationView.java:720)
    at com.instabug.library.c.onClick (InstabugAnnotationFragment.java:220)
    at android.view.View.performClick (View.java:5697)
    at android.view.View$PerformClick.run (View.java:22526)
    at android.os.Handler.handleCallback (Handler.java:739)
    at android.os.Handler.dispatchMessage (Handler.java:95)
    at android.os.Looper.loop (Looper.java:158)
    at android.app.ActivityThread.main (ActivityThread.java:7225)
    at java.lang.reflect.Method.invoke (Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1230)
    at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1120)
```

Please update me if theres a new release soon with a fix.

Best Regards.

**Comment by tarek360** [comment]
Hi @joeabouassi ,
Thank you for reporting this, I will check it but I have a question, does it happen consistently with you?

**Comment by joeabouassi** [comment]
Hi @tarek360 ,

It happened 19 hours ago for the first time as reported by Instabug in one of my apps, but Im not sure if it happens often because as I was testing Instabug crash reporting sometimes the crashes don't get uploaded.

**Comment by tarek360** [comment]
Regarding the crash reporting on the dashboard please contact us on [our website](https://instabug.com/) and we will check it for you now, Thank you @joeabouassi

**Comment by joeabouassi** [comment]
Sure will do!
Please let me know if u manage to pin point and fix it.

Thank you @tarek360

**Comment by NadineMansour** [comment]
@joeabouassi We fixed the issue that you reported. It’ll be released soon. I’ll let you know once it goes live

**Comment by joeabouassi** [comment]
Thank you for the update @NadineMansour !

**Comment by NadineMansour** [comment]
@joeabouassi The fix is a632e790983bb11edbdcb129436f6e583f964bd6 was released in the latest version 4.2.10.

**Comment by NadineMansour** [comment]
@joeabouassi I will close this issue for now. 
Please, feel free to reopen it again if you have any further concerns.
